orchagent/bufferorch.cpp

```diff
@@ -58,9 +58,20 @@ void BufferOrch::initBufferReadyList(Table& table)
 {
     SWSS_LOG_ENTER();
```

Comment:
This function will be called twice, so m_port_ready_list_ref will be overwritten again. Is it a bug?

Reply:
I try to fix this in https://github.com/Azure/sonic-swss/pull/590, please help review.